easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.3-2023b.yml

```diff
@@ -0,0 +1,5 @@
+easyconfigs:
+  - LAMMPS-29Aug2024-foss-2023b-kokkos.eb:
+      # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21436
+      options:
+        from-pr: 21436
```

Comment:
@laraPPr Please use `--from-commit` rather than `--from-pr` (in this case, last commit in PR, so `9dc24e57880a8adb06ae10557c5315e66671a533`)